internal/connector/test/integration/features/connector-multitenancy-api.feature

```diff
@@ -456,6 +456,57 @@ Feature: connector namespaces API
     And the response code should be 200
     And the ".total" selection from the response should match "1"
 
+    # check that namespace name is generated
+    Given I am logged in as "Dusty"
+    When I POST path "/v1/kafka_connector_namespaces/" with json body:
+    """
+    {
+      "cluster_id": "${connector_cluster_id}",
+      "kind": "organisation",
+      "annotations": [
+        {
+          "key": "connector_mgmt.bf2.org/profile",
+          "value": "default-profile"
```

Comment:
unrelated to this PR but haven't we switched to simple key-val pairs for annotations ? similar to https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/openapi/kas-fleet-manager-private.yaml#L240-L249

Reply:
Not yet, will do when we have sorted out all the higher priority functional issues. Or most likely post preview release.